### PR TITLE
Initial draft of blog post.

### PR DESCRIPTION
**Pull request by ksbeattie**
This will need to have the img src url fixed after the [PR in the image repo](https://github.com/betterscientificsoftware/images/pull/7) is merged.

fyi: @dangunter

### CODE REVIEW

**Comment by bernhold** [comment]
@karbarz can you please take a pass through this blog article to get it into our format, etc?

Matt Stone is working on the hero image to go with it.

**Comment by karbarz** [comment]
Standard edits for clarity, fixed spacing issues, checked all links. 

Should the IEEE glossary be a link? 
10.1109/IEEESTD.1990.101064 or https://ieeexplore.ieee.org/document/159342

@bernhold @ksbeattie

**Review by rinkug — APPROVED** [review]

**Comment by gpieper** [comment]
Ok, I just committed the changes. I had a lot of comments, so I thank you for letting me see this. If the authors have questions, I would be happy to discuss my suggeestions. I do hope I have not offended anyone.
Gail

**Review by karbarz — APPROVED** [review]
Reviewed Gail's changes - all looks good.
There are a couple of misspellings in this file, however, that don't show up when viewed in the editing section. Just verifying that these do not need to be fixed. Examples are: 

"aree in four other states."
"Tge goal:  developer productivity"

File: Articles/Blog/2021-03-useful-practices-for-SEoMsDSP.md [hunk 1]
```diff
@@ -168,7 +158,7 @@ understanding and addressing the unique challenges in leading multi-institutiona
 dispersed scientific software development teams while still producing effective and usable software,
 particularly teams composed of members from scientific but not necessarily software engineering
 backgrounds.  He has worked in industry as a software engineer and release manager and has been at LBNL for  the past 20
-years.  He also tortures local music venue attendees playing bass in rock bands.
+years.  He also tours local music venue attendees playing bass in rock bands.
```
Comment (ksbeattie) [inline Articles/Blog/2021-03-useful-practices-for-SEoMsDSP.md]: Maybe this is too much of my odd sense of humor?  Would "torments" be better?

Comment (rinkug) [reply Articles/Blog/2021-03-useful-practices-for-SEoMsDSP.md]: > Maybe this is too much of my odd sense of humor? Would "torments" be better?

:)) Either is ok.

**Review by rinkug — APPROVED** [review]

**Pull request merged**